index.html

```diff
@@ -824,7 +833,297 @@ <h3>
+        representation that is capable of preserving all properties, 
+        all properties SHOULD be preserved.
+      </p>
+
+      <p>For example, preserving <code>@context</code> in YAML, 
+        allows for the document to later be used with 
+        <a href="https://www.w3.org/TR/json-ld11-api/#loaddocumentcallback">documentLoaders</a> that use 
+        <a href="https://www.w3.org/TR/json-ld11-framing/">frame</a> to perform dereferencing.
+      </p>  
+
+      <pre class="example">
+---
+"@context":
+- https://www.w3.org/ns/did/v1
+- https://w3id.org/security/suites/jws-2020/v1
+id: did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+verificationMethod:
+- id: did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6#z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+  type: JsonWebKey2020
+  controller: did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+  publicKeyJwk:
+    kty: EC
+    crv: P-384
+    x: ELYfgFzSi43VGgKlyuYBKeoW00DmNqf5VuMWD1iUTTHQ8NRQrR4KHShzH_DrvOhh
+    y: sftqdHdso4JPQO3MLnsXmWbAFqjmHLgP8H_DZSLxx3ei9kLj_W3R6NhtH03bwE1C
+assertionMethod:
+- did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6#z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+authentication:
+- did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6#z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+capabilityInvocation:
+- did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6#z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+capabilityDelegation:
+- did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6#z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+keyAgreement:
+- did:key:z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6#z82Lkm8gN2PGD7WbUiTUKRHgSptPhGPAQG9HT9HybYE9LoKYwcikh5rAU9gGo3qTne3ZHf6
+
+      </pre>
+
+      <p class="note">Preserving members across representations is also common in 
+        <a href="https://swagger.io/specification/">OpenAPI Specification 3.0.3</a>,
+      which supports 
+      <a href="https://datatracker.ietf.org/doc/html/draft-wright-json-schema-00#section-4.2">JSON Schema</a>
+      represented as YAML or JSON.
+      </p>
+
+      <p>DID Methods that support both <code>application/did+ld+json</code> and <code>application/did+json</code> 
+        SHOULD return <code>application/did+json</code> with an <code>@context</code>, because it enables
+        both representations to be used with tooling and standards that support semantics (such as [[VC-DATA-MODEL]]), 
+        and because the default behavior of JSON processors is to ignore object members that are not understood.
+      </p>
+
+      <p>DID Document representations produced by an implementation SHOULD be treated as immutable, 
```

Comment:
Hmm... SHOULD is normative guidance... but this is a NOTE (supposed to be entirely non-normative)... is this a mistake or are we trying to put normative language in this document?

Reply:
agree - this should stick to non-normative conventions typical for a NOTE